bin/unlink

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/perl
+
+=begin metadata
+
+Name: unlink
+Description: simpler than rm
+Author: Michael Mikonos
+License:
+
+=end metadata
+
+=cut
+
+use strict;
+
+use File::Basename qw(basename);
+use Getopt::Std qw(getopts);
+
+use constant EX_SUCCESS => 0;
+use constant EX_FAILURE => 1;
+
+my $Program = basename($0);
+
+unless (getopts('')) {
+    warn "usage: $Program FILE\n";
+    exit EX_FAILURE;
+}
+unless (@ARGV) {
+    warn "$Program: missing operand\n";
+    exit EX_FAILURE;
+}
+my $file = shift;
+if (@ARGV) {
+    warn "$Program: extra operand: '$ARGV[0]'\n";
+    exit EX_FAILURE;
+}
+if (-d $file) {
+    warn "$Program: cannot unlink '$file': is a directory\n";
+    exit EX_FAILURE;
+}
+unless (unlink $file) {
+    warn "$Program: cannot unlink '$file': $!\n";
+    exit EX_FAILURE;
+}
+exit EX_SUCCESS;
+
+__END__
+
+=pod
```

Comment:
Add =encoding utf8 to start pod. This is simply something I'm trying to make consistent everywhere.